cmd/state/state.go

```diff
@@ -164,10 +173,11 @@ func GetDefaultFlags(homeDir string) GlobalFlags {
 		ProfilingEnabled: false,
 		ConfigFilePath:   filepath.Join(homeDir, "k6", defaultConfigFileName),
 		LogOutput:        "stderr",
+		BuildServiceURL:  "https://ingest.k6.io/builder/api/v1",
```

Comment:
Nit; Can we define this as a `const`, like `defaultConfigFileName`, please? So, this way we have the consts that belong to the default flags together.